mitodl/smoot-design
# Initial set up

### What are the relevant tickets?
For https://github.com/mitodl/hq/issues/6039

### Description (What does it do?)


### Screenshots (if appropriate):
Migrates ThemeProvider + theme, Button, Typography.

### How can this be tested?
1. Try out components on docs site: https://mitodl.github.io/smoot-design
    - In particular, please read https://mitodl.github.io/smoot-design/?path=/docs/smoot-design-themeprovider--docs#custom-link-adapter
2. Install locally with `yarn install`. Some things to check:
    - `pre-commit install` followed by `pre-commit run --all` produces no errors
    - introducing a linting error produces linting errors
3. Run `yarn storybook` locally. (Or `yarn storybook --docs`). Test out storybook locally.

### Additional Context
